### PR TITLE
UI Unit test additions, fixes, and build updates

### PR DESCRIPTION
**Pull request by cdcabrera**
* Unit tests added for reducers
* Updated app unit test to do a quick render.
* Quick fix for limited support of the "finally" Promise method as related to Fetch
* Service worker moved
* Build and app dependency updates
* Convert content and vertical nav component import

updates #434

### CODE REVIEW

File: client/src/redux/actions/credentialsActions.js [hunk 1]
```diff
@@ -24,11 +24,11 @@ const getCredentials = () => {
       .getCredentials()
       .then(success => {
         dispatch(getCredentialsSuccess(success));
+        dispatch(getCredentialsLoading(false));
       })
       .catch(error => {
         dispatch(getCredentialsError(true, error.message));
```
Comment (jeff-phillips-18) [inline client/src/redux/actions/credentialsActions.js]: Should `dispatch(getCredentialsLoading(false));` here as well.

File: client/src/redux/actions/factsActions.js [hunk 2]
```diff
@@ -24,11 +24,11 @@ const addFacts = () => {
       .addFacts()
       .then(success => {
         dispatch(addFactsSuccess(success));
+        dispatch(addFactsLoading(false));
       })
       .catch(error => {
         dispatch(addFactsError(true, error.message));
-      })
```
Comment (jeff-phillips-18) [inline client/src/redux/actions/factsActions.js]: Ditto.

File: client/src/redux/actions/reportsActions.js [hunk 3]
```diff
@@ -24,11 +24,11 @@ const getReports = () => {
       .getReports()
       .then(success => {
         dispatch(getReportsSuccess(success));
+        dispatch(getReportsLoading(false));
       })
       .catch(error => {
         dispatch(getReportsError(true, error.message));
```
Comment (jeff-phillips-18) [inline client/src/redux/actions/reportsActions.js]: and here...

File: client/src/redux/actions/sourcesActions.js [hunk 4]
```diff
@@ -24,11 +24,11 @@ const getSources = () => {
       .getSources()
       .then(success => {
         dispatch(getSourcesSuccess(success));
+        dispatch(getSourcesLoading(false));
       })
       .catch(error => {
         dispatch(getSourcesError(true, error.message));
```
Comment (jeff-phillips-18) [inline client/src/redux/actions/sourcesActions.js]: enough said.

File: client/src/redux/actions/credentialsActions.js (hunk 1 above)
Comment (cdcabrera) [reply client/src/redux/actions/credentialsActions.js]: Indeed +1

**Pull request merged**